### ISSUE TITLE
QC test message should specify that they are "real time"

### ISSUE THREAD

**Issue by richardsc** [ArgoCanada/argoFloats#546]
Noticed this in Jaimie's (great!) argoFloats workshop, where she made a point of highlighting the difference between "bad" flags for DM data despite the message that says it passed all tests, e.g.

```
Float ID 5904811 is flagged bad.
Tests performed: 2 3 4 5 6 13 14 15 16 17 19
     Passed all tests
```

Maybe it should specify by saying e.g. "Passed all Real-time tests"?

**Comment by dankelley** [comment]
I agree.  I'll assign "the boss" to this.

**Comment by j-harbin** [comment]
Fixed in commit 2af50302eb739bd83dee23bee52b13a5c9b721f4 of develop.
